index.html

```diff
@@ -624,13 +629,13 @@ <h3>Synchronization of text track cue rendering</h3>
         </p>
         <p>
           The <a>time marches on</a> steps in [[HTML]] control the firing of cue
-          events during media playback. <a>Time marches on</a> requires a
-          <code>timeupdate</code> event to be fired at the
```

Comment:
How about rewording as:

```
<a>Time marches on</a> requires the <a>timeupdate</a> event to be fired
from time to time during media playback, and step 6 of that algorithm includes the note: 
"The event thus is not to be fired faster than about 66Hz or slower than 4Hz
(assuming the event handlers don't take longer than 250ms to run)."  In practice it
...
```

and then continue with this pull requests content from line 636 onwards?